poi_fr.md

```diff
@@ -0,0 +1,211 @@
+
+### poi_type.txt (requis)
+
+Ce fichier contient la liste des types de POI de l'export.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+`poi_type_id` | chaine | Requis | Identifiant unique et pérenne de la catégorie de POI
+`poi_type_name` | chaine | Requis | Nom de la catégorie de POI
+
+
+Le nombre de catégories et le contenu du champ `poi_type_id` est libre, mais
+des contraintes sont associées à l'utilisation dans Navitia et dans le NMP.
+
+
+### poi.txt (requis)
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+`poi_id` | chaine | Requis | Identifiant unique et pérenne du POI
+`poi_type_id` | chaine | Requis | Identifiant du type de POI (lien vers le fichier [`poi_type.txt`](#poi_typetxt-requis))
+`poi_name` | chaine | Requis | Nom du POI
+`poi_lat` | décimal | Requis | Latitude du POI
+`poi_lon` | décimal | Requis | Longitude du POI
+`poi_weight` | entier | Requis | Poids associé au POI dans la reconnaissance de la saisie (API "places"). Plus le poids est grand, plus le POI remontera en haut de la liste de la recherche (pour des POI avec un matching identique). 
+`poi_visible` | entier | Requis | Indique si le POI doit être présenté dans l'auto-complétion.
+
+L'identifiant du poi `poi_id` est unique toutes sources de POI confondus. Il
+convient donc d'être vigilent en cas de source multiple de POI pour éviter
+qu'il y ait collision.
+
+### poi_properties.txt (optionnel)
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+`poi_id` | chaine | Requis | Identifiant du POI (lien vers le fichier [`poi.txt`](#poitxt-requis))
+`key` | chaine | Requis | Nom de la propriété du POI
+`value` | chaine | Requis | Valeur associée à la propriété du POI
+
+Le couple (`poi_id`, `key`) est unique.
+
+La liste des propriétés des POI n'est pas fixée ni limitée, mais certaines
+propriétés ont des effets et donc des contraintes particulières dans Navitia et
+dans les médias.
+
+# Les propriétés "transport"
+
+Certaines propriétés ont un impact sur les itinéraires proposés par Navitia.
+Propriété | Clé | Valeur (si fixée) | Commentaire
+--- | --- | --- | ---
+Rabattement en vélo | `bike_access_point` | 0 ou 1 | Indique si le POI (ou l'arrêt) est un point de rabattement en vélo vers le transport en commun. Si la propriété est absente, il est considéré que le POI n'est pas un point de rabattement. Cette propriété n'est pas utilisée par le moteur à ce jour. Elle est prévue en cas de besoin ulterieur.
+Rabattement en voiture | `car_access_point` | 0 ou 1 | Indique si le POI (ou l'arrêt) est un point de rabattement en voiture vers le transport en commun (ie. un parking ou un P+R). Si la propriété est absente, il est considéré que le POI n'est pas un point de rabattement.
+
+# Liste standard des propriétés
+
+La liste des propriétés supplémentaires sur un POI n'est pas fixe. Cependant,
+pour permettre un affichage standardisé sur les produits Canal TP, les
+propriétés utilisées sont normalisées.
```

Comment:
```suggestion
La liste des propriétés supplémentaires sur un POI n'est pas fixe. Cependant,
pour permettre un affichage standardisé sur les produits Kisio Digital, les
propriétés utilisées sont normalisées.
```

Reply:
Done in 961ed4f180b20ef4c1c35f94d18ce6d45b98279a.